### ISSUE TITLE
how to open ssh

### ISSUE THREAD

**Issue by Ali-Qiasi** [MichaelBosello/carla-colab#12]
Hi. how can i open a ssh connection?
I tryed to download and install Cloudflared but that file does not install. its just a command page.

**Comment by MichaelBosello** [comment]
HI. Can you elaborate more on your question? Do you have problems installing Cloudflare? Have you seen the executables in [this page](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/install-and-setup/installation)?

**Comment by Ali-Qiasi** [comment]
yes its exactly i want. after download the app i can install and open a ssh connection to colab. its just a prompt page than cant use any command. i used window 11 x64

**Comment by Ali-Qiasi** [comment]
Can you help me with a video? or something else
I do that you said but that is not work. 
I run Turbo VNC and see this massage:
"Failed to connect to server (localhost:1)"
